libqtile/widget/battery.py

```diff
@@ -550,13 +553,13 @@ def build_string(self, status: BatteryStatus) -> str:
             char = self.discharge_char
         elif status.state == BatteryState.FULL:
             if self.show_short_text:
-                return "Full"
+                return self.full_short_text
             char = self.full_char
         elif status.state == BatteryState.EMPTY or (
             status.state == BatteryState.UNKNOWN and status.percent == 0
         ):
             if self.show_short_text:
-                return "Empty"
+                return self.full_empty_text
```

Comment:
This should be `empty_short_text` right?

Reply:
Yep, fixed in latest revision.

Reply:
Thanks!